libbeat/idxmgmt/lifecycle/config.go

```diff
@@ -33,9 +33,9 @@ type Config struct {
 	// used only for testing
 	policyRaw *Policy
 
-	// CheckExists can disable the check for an existing policy. Check required
-	// read_ilm privileges.  If check is disabled the policy will only be
-	// installed if Overwrite is enabled.
+	// CheckExists can disable the check for an existing policy. This check
+	// requires read_ilm privileges. If CheckExists is disabled the policy
```

Comment:
Nit: additional clarity.

```suggestion
	// requires read_ilm privileges. If CheckExists is set to false, the policy
```